state/indexer/sink/psql/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TYPE block_event_type AS ENUM ('begin_block', 'end_block', '');
+CREATE TABLE block_events (
+    block_event_id SERIAL PRIMARY KEY,
+    key VARCHAR NOT NULL,
+    value VARCHAR NOT NULL,
+    height INTEGER NOT NULL,
+    type block_event_type
+);
+CREATE TABLE tx_results (
+tx_result_id SERIAL PRIMARY KEY,
```

Comment:
```suggestion
id SERIAL PRIMARY KEY,
```

Reply:
If I use `id` to the table's Primary key, 
I will get an error `pq: column reference "id" is ambiguous` when doing the query.